src/AnyOf.ts

```diff
@@ -12,13 +12,13 @@ export class AnyOf {
         return (
             item?.type === beaversSystemInterface.configLootItemType && (
                 item?.system?.source === Settings.ANYOF_SUBTYPE ||
-                item?.flags["beavers-crafting"]?.subtype === Settings.ANYOF_SUBTYPE
+                getProperty(item,`flags.beavers-crafting.subtype`) === Settings.ANYOF_SUBTYPE
```

Comment:
First of thx for your pullrequest, got 2 things i wasn't aware of the backticks ecma6 syntax for once and that foundry has already a getProperty and setProperty helper method.

But i am still unsure  what had been the problem with all those places, but yes we can improve them if you like.